spec/jobs/approved_file_copy_job_spec.rb

```diff
@@ -4,41 +4,64 @@
 RSpec.describe ApprovedFileMoveJob, type: :job do
   include ActiveJob::TestHelper
 
-  subject(:job) do
-    described_class.perform_later(work_id: work.id, source_bucket: "example-bucket", source_key: "10.34770/ackh-7y71/1/test_key", target_bucket: "example-bucket-post",
-                                  target_key: "10.34770/ackh-7y71/1/test_key", size: 200)
-  end
-  let(:fake_s3_service) { stub_s3 }
-  let(:work) { FactoryBot.create :approved_work }
-
-  before do
-    fake_completion = instance_double(Seahorse::Client::Response, "successful?": true)
-    allow(fake_s3_service).to receive(:copy_file).and_return(fake_completion)
-    allow(fake_s3_service).to receive(:delete_s3_object).and_return(fake_completion)
-    allow(fake_s3_service).to receive(:check_file).and_return(true)
-    allow(fake_s3_service.client).to receive(:put_object).and_return(nil)
-  end
-
-  it "runs an aws copy and delete" do
-    perform_enqueued_jobs { job }
-    expect(fake_s3_service).to have_received(:copy_file).with(size: 200, source_key: "/example-bucket/10.34770/ackh-7y71/1/test_key",
-                                                              target_bucket: "example-bucket-post", target_key: "10.34770/ackh-7y71/1/test_key")
-    expect(fake_s3_service).to have_received(:delete_s3_object).with("10.34770/ackh-7y71/1/test_key", bucket: "example-bucket")
-    expect(fake_s3_service).to have_received(:delete_s3_object).with(work.s3_object_key, bucket: "example-bucket")
-    expect(fake_s3_service.client).to have_received(:put_object).with({ bucket: "example-bucket-post", content_length: 0, key: "#{work.doi}/#{work.id}/princeton_data_commons/" })
-  end
+  describe "approval process" do
+    subject(:job) do
+      described_class.perform_later(work_id: work.id, source_bucket: "example-bucket", source_key: "10.34770/ackh-7y71/1/test_key", target_bucket: "example-bucket-post",
+                                    target_key: "10.34770/ackh-7y71/1/test_key", size: 200)
+    end
+    let(:fake_s3_service) { stub_s3 }
+    let(:work) { FactoryBot.create :approved_work }
 
-  context "the copy fails" do
     before do
-      fake_completion = instance_double(Seahorse::Client::Response, "successful?": false)
+      fake_completion = instance_double(Seahorse::Client::Response, "successful?": true)
```

Comment:
This diff is misleading. I split the test in two in order to pass different buckets, but the the way the diff renders it looks like I changed the last value in the call to `instance_double()` from `false` to `true` but I didn't, if you look at like 15 in the diff you see that it was `true` before.